config/kubernetes-client/org.yaml

```diff
@@ -5,6 +5,7 @@ has_organization_projects: true
 has_repository_projects: true
 members_can_create_repositories: false
 admins:
+- brendandburns
```

Comment:
Only the GitHub admin team can be in the `admins` list: https://github.com/kubernetes/community/tree/master/github-management#github-administration-team :grimacing: 

@brendandburns if **org** admin permissions are needed for setting up something, could you open an issue on this repo? The github admin team can handle that. :)

In general,  you should have admin permissions across all repos....except the python ones. Could you leave only the python change in this PR? Thanks!

Reply:
@nikhita I filed https://github.com/kubernetes/org/issues/635

Thanks.